docs/Developers/Newcomers_Guide.rst

```diff
@@ -383,6 +383,21 @@ and clicking ``New Pull Request``.
 
 **Congratulations!** You have now created your first ``Pull Request``!
 
+Now, before having other contributors review your work, the system will do an
+automated code review and will comment on your Pull Request.
+You may be required to enhance your Pull Request by modifying your branch on
+your fork.
+
+.. important::
+
+    It is **never** necessary to close your Pull Request. **Never!**.
+    It is **always** possible to modify your pull request using ``git``.
+    You must not delete your fork, delete your branch, close your
+    ``Pull Request`` or start a new one etc. All of those actions will create a
+    mess. If instead of learning to use ``git``, you create a second
+    ``Pull Request`` on the same issue, a maintainer will close your second PR
```

Comment:
PR -> Pull Request. Just to keep them consistent.

Reply:
Alright, I will try rebasing first, and then I will change that as well.